templates/vue/cypress/support/roles.js

```diff
@@ -0,0 +1,10 @@
+export default {
+  admin: {
+    username: "admin",
+    password: "Tapestry123!",
+  },
+  subscriber: {
+    username: "narendras",
+    password: ")%AbNZgppS)uNvxT5)OdgzhX",
```

Comment:
If these require set up from the user's side, please add it to the README file. Please do not use your name, but instead a more generic username